prep-css-display/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<head>
+  <title>CSS Display Properties</title>
+  <style>
+</head>
```

Comment:
The closing tag for the head element should go after the closing tag for the style element.